src/style.css

```diff
@@ -0,0 +1,128 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-size: 14px;
+  box-sizing: border-box;
+}
+
+/* #region form */
+.big__form {
+  position: relative;
+}
+
+.small__form {
+  position: relative;
+}
+/* #endregion */
+
+/* #region image */
+.image {
+  position: absolute;
+  top: 25px;
+  left: 26px;
+  height: 19px;
+  color: #687c94;
```

Comment:
https://www.w3schools.com/howto/howto_css_center-vertical.asp Use the way from this article to center your image vertically, now it's not centered properly